### PR TITLE
Use `import numpy as np` everywhere

### PR DESCRIPTION
**Pull request by eumiro**
In order to have a consistent import structure (and to discover deprecations like in #862 more easily) , replace all `import numpy` with `import numpy as np` and the `numpy.` with `np.` in the code.

Two usages of `np` as a variable have been replaced by `arr`.

### CODE REVIEW

**Review by avalentino — APPROVED** [review]

File: tables/tests/test_array.py [hunk 1]
```diff
@@ -1520,15 +1519,15 @@ class GI1NACloseTestCase(GI1NATestCase):
...
     # A more complex example
     title = "Rank-1,2 case 2"
-    numericalList = numpy.array([3, 4])
-    numericalListME = numpy.array([[3, 2, 1, 0, 4, 5, 6],
+    numericalList = np.array([3, 4])
+    numericalListME = np.array([[3, 2, 1, 0, 4, 5, 6],
                                    [2, 1, 0, 4, 5, 6, 7],
                                    [4, 3, 2, 1, 0, 4, 5],
                                    [3, 2, 1, 0, 4, 5, 6],
                                    [3, 2, 1, 0, 4, 5, 6]])
```
Comment (keszybz) [inline tables/tests/test_array.py:1527]: indentation...

File: tables/tests/test_array.py [hunk 2]
```diff
@@ -1834,15 +1833,15 @@ class SI1NACloseTestCase(SI1NATestCase):
...
```
Comment (keszybz) [inline tables/tests/test_array.py:1841]: here too

File: tables/tests/test_array.py [hunk 3]
```diff
@@ -1967,15 +1966,15 @@ class GE1NACloseTestCase(GE1NATestCase):
...
```
Comment (keszybz) [inline tables/tests/test_array.py:1974]: ditto

File: tables/tests/test_create.py [hunk 4]
```diff
@@ -2434,18 +2434,18 @@ class QuantizeTestCase(common.TempFileMixin, TestCase):
     def setUp(self):
         super().setUp()
 
-        self.data = numpy.linspace(-5., 5., 41)
-        self.randomdata = numpy.random.random_sample(1_000_000)
-        self.randomints = numpy.random.randint(-1_000_000, 1_000_000,
+        self.data = np.linspace(-5., 5., 41)
+        self.randomdata = np.random.random_sample(1_000_000)
+        self.randomints = np.random.randint(-1_000_000, 1_000_000,
                                                1_000_000).astype('int64')
```
Comment (keszybz) [inline tables/tests/test_create.py:2440]: ditto

File: tables/tests/test_create.py [hunk 5]
```diff
@@ -2503,22 +2503,22 @@ def test01_quantizeDataMaxError(self):
...
+        np.testing.assert_array_equal(self.h5file.root.data1[:], self.data)
+        np.testing.assert_array_equal(self.h5file.root.data2[:], self.data)
+        np.testing.assert_array_equal(self.h5file.root.data0[:],
                                          self.quantizeddata_0)
-        numpy.testing.assert_array_equal(self.h5file.root.datam1[:],
+        np.testing.assert_array_equal(self.h5file.root.datam1[:],
                                          self.quantizeddata_m1)
-        numpy.testing.assert_array_equal(self.h5file.root.integers[:],
+        np.testing.assert_array_equal(self.h5file.root.integers[:],
                                          self.randomints)
```
Comment (keszybz) [inline tables/tests/test_create.py:2522]: here too

File: tables/tests/test_earray.py [hunk 6]
```diff
@@ -412,11 +412,11 @@ def test03_readEArray_out_argument(self):
         earray.nrowsinbuf = 3
         # Build the array to do comparisons
         if self.type == "string":
-            object_ = numpy.ndarray(buffer=b"a"*self.objsize,
+            object_ = np.ndarray(buffer=b"a"*self.objsize,
                                     shape=self.rowshape,
                                     dtype="S%s" % earray.atom.itemsize)
```
Comment (keszybz) [inline tables/tests/test_earray.py:417]: ...

File: tables/tests/test_earray.py [hunk 7]
```diff
@@ -534,11 +534,11 @@ def test04_getitemEArray(self):
 
         # Build the array to do comparisons
         if self.type == "string":
-            object_ = numpy.ndarray(buffer=b"a"*self.objsize,
+            object_ = np.ndarray(buffer=b"a"*self.objsize,
                                     shape=self.rowshape,
                                     dtype="S%s" % earray.atom.itemsize)
```
Comment (keszybz) [inline tables/tests/test_earray.py:539]: ...

File: tables/tests/test_hdf5compat.py [hunk 8]
```diff
@@ -353,8 +353,8 @@ def test_ref_utf_str(self):
 
         self.assertTrue(common.areArraysEqual(
                         array[0][0][0],
-                        numpy.array([0, 0],
-                                    dtype=numpy.uint64)))
+                        np.array([0, 0],
+                                    dtype=np.uint64)))
```
Comment (keszybz) [inline tables/tests/test_hdf5compat.py:357]: ...

File: tables/tests/test_hdf5compat.py [hunk 9]
```diff
@@ -369,15 +369,15 @@ def test_ref_str(self):
...
+                                    dtype=np.uint16)))
 
     def test_double_ref(self):
         array = self.h5file.get_node('/var')
         self.assertTrue(common.areArraysEqual(
                         array[2][0][0][1][0],
-                        numpy.array([[105], [110], [115], [105], [100], [101]],
-                                    dtype=numpy.uint16)))
+                        np.array([[105], [110], [115], [105], [100], [101]],
+                                    dtype=np.uint16)))
```
Comment (keszybz) [inline tables/tests/test_hdf5compat.py:380]: ...

**Review by keszybz — APPROVED** [review]
LGTM. The only thing is the indentation, which might be worth fixing but might not.

**Comment by eumiro** [comment]
> LGTM. The only thing is the indentation, which might be worth fixing but might not.

You're absolutely right, @keszybz. I believe the whole codebase needs one single big `black` slap and then some sort of `pre-commit` that checks all further commits. For now I'll fix those diffs.

**Pull request merged**